pkg/segment/structs/segstructs.go

```diff
@@ -485,7 +485,7 @@ type Progress struct {
 type NodeResult struct {
 	AllRecords                  []*utils.RecordResultContainer
 	ErrList                     []error                     // Need to eventually replace ErrList with GlobalSearchErrors to prevent duplicate errors
-	GlobalSearchErrors          map[string]*SearchErrorInfo // maps global error from error message -> error info
+	GlobalSearchErrors          map[string]*SearchErrorInfo // maps global error from error message -> error info  (Deprecated: use toputils.BatchError)
```

Comment:
Are you going to delete this soon in a different PR? Or do we need to wait for something before we can fully remove this?

Reply:
Yeah to delete this, we need to do more work. I will do that in a later PR.